app/tasks.py

```diff
@@ -379,3 +380,65 @@ def make_query(self):
     def update_counter(self, counter, person):
         person.update_index(['old', 'new'])
         person.put()
+
+
+class NotifyBadReviewStatus(utils.BaseHandler):
+    """This task sends email notification when the number of unreviewed notes
+    exceeds threshold.
+    """
+    ACTION = 'tasks/notify_bad_review_status'
+
+    def task_name(self):
+        return 'notify-bad-review-status'
+
+    def get(self):
+        if self.repo:
+            try:
+                count_of_unreviewed_notes = \
```

Comment:
It is preferred to use brackets instead of backslashes for line breaks:
https://google.github.io/styleguide/pyguide.html#Line_length

e.g.,
```python
count_of_unreviewed_notes = (
    model.Note.unreviewed_record_count(self.repo))
```

I know that there are many places using backslashes in our code base :(